src/java/uk/ac/rhul/cs/cl1/ui/cytoscape3/ExtractClusterAction.java

```diff
@@ -43,23 +46,37 @@ public void actionPerformed(ActionEvent event) {
 		}
 		
 		resultViewer.selectNodes(selectedNodes);
+		NewNetworkSelectedNodesAndEdgesTaskFactory taskFactory = 
+                                app.getService(CySwingAppAdapter.class).get_NewNetworkSelectedNodesAndEdgesTaskFactory();
 		
-		NewNetworkSelectedNodesAndEdgesTaskFatory taskFactory =
-				this.resultViewer.getCytoscapeApp().getService(NewNetworkSelectedNodesAndEdgesTaskFatory.class);
```

Comment:
Okay, I'm pretty sure that this was a typo in the Cytoscape 3.0.0-beta3 API package that I used back in the old days to compile the Cytoscape 3 plugin. I see that the name of this interface was fixed in the Cytoscape 3.0.0 API.